main.py

```diff
@@ -1,7 +1,26 @@
 import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
 
-from django.db.models import QuerySet
 
+def main() -> None:
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
 
-def main() -> QuerySet:
-    pass
+    Actor.objects.create(first_name="George", last_name="Klooney")
```

Comment:
The last name 'Klooney' seems to be a typo. It is corrected later in the code, but it's better to use the correct spelling 'Clooney' from the beginning.